draft-ldbc-cats-framework.md

```diff
@@ -230,11 +230,41 @@ CATS Service ID (CS-ID):
 CATS Instance Selector ID (CIS-ID):
 : An identifier of a specific service contact instance.
 
-## CATS Components {#sec-cat-arch}
+## CATS Framework {#sec-cats-framework}
+
+The high-level CATS framework can be illustrated in Figure 1.  Starting from the bottom part of the figure and moving towards the upper part, the following planes are defined.
+
+~~~                           
+   +----------------------------------+  |         +--------+
+   |         Management Plane         |  |  Intf   |        |
+   +----------------------------------+  |<=======>| C-SMA  |
+   |           Control Plane          |  |         |        |
+   +----------------------------------+  |         +---+----+
+                   /\ SBI                |             |    
```

Comment:
```suggestion
                   /\ NFI                |             |    
```

Reply:
Let's avoid geocoordinate things and use meaningful terms. NFI (Network Facing Interface)

Reply:
LGTM, let's merge and see what people think. But we might need to define what is NFI?